api/prisma/migrations/20210908091553_bddchange/migration.sql

```diff
@@ -0,0 +1,22 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `currentPlayer` on the `Game` table. All the data in the column will be lost.
+  - You are about to drop the column `directionSelected` on the `Game` table. All the data in the column will be lost.
+  - You are about to drop the column `graph` on the `Game` table. All the data in the column will be lost.
+  - You are about to drop the column `hasWinner` on the `Game` table. All the data in the column will be lost.
+  - You are about to drop the column `marbleClicked` on the `Game` table. All the data in the column will be lost.
+  - You are about to drop the column `players` on the `Game` table. All the data in the column will be lost.
+  - You are about to drop the column `started` on the `Game` table. All the data in the column will be lost.
+  - Added the required column `name` to the `Game` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "Game" DROP COLUMN "currentPlayer",
```

Comment:
As this is first time migrations are introduced (and not yet deployed in production I assume), you should update the init migration and remove this one. This means you'll have to reset your local environment though

Reply:
Yes, i'll clear the migrations before the deployment

Reply:
this has been done in the last push